test/helpers/common.bash

```
@@ -0,0 +1,22 @@
+    echo "ERROR: Source file /workspace/bin/git-shiplog does not exist!" >&2
+    return 1
+  fi
+
+  if [[ ! -x /workspace/bin/git-shiplog ]]; then
+    echo "ERROR: Source file /workspace/bin/git-shiplog is not executable!" >&2
+    return 1
+  fi
+
+  install -m 0755 /workspace/bin/git-shiplog /usr/local/bin/git-shiplog
+
+  if [[ $? -ne 0 ]]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)